easy_my_coop_website/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "depends": [
         "easy_my_coop",
         "website",
-        "website_recaptcha_reloaded",
+        "website_recaptcha",
```

Comment:
```suggestion
        "website_form_recaptcha",
```

Is this a drop-in replacement?

Reply:
Not sure what a drop-in replacement is. We do not use another module, we just move it to addons, and rename it.

Reply:
A drop-in replacement is when you swap one component for another without any troubles :)

I don't think this is going to work, though. I mean, it will _work_, but this will be a blocker for moving this module to the OCA. We want to depend exclusively on OCA modules, not on external modules. Pinging @robinkeunen in case I'm mistaken.

Reply:
Fixed (see [this PR](https://github.com/coopiteasy/addons/pull/216))